src/tooltipServiceWrapper.ts

```diff
@@ -97,53 +62,57 @@ class TooltipServiceWrapper implements ITooltipServiceWrapper {
         // Mouse events
         selection.on("mouseover.tooltip", () => {
             // Ignore mouseover while handling touch events
-            if (!this.canDisplayTooltip(event)) {
+            if (!this.canDisplayTooltip(d3Event))
                 return;
-            }
 
             let tooltipEventArgs = this.makeTooltipEventArgs<T>(rootNode, true, false);
-            if (!tooltipEventArgs) { return; }
+            if (!tooltipEventArgs)
+                return;
 
             let tooltipInfo = getTooltipInfoDelegate(tooltipEventArgs);
-            if (tooltipInfo == null) { return; }
+            if (tooltipInfo == null)
+                return;
 
             let selectionId = getDataPointIdentity(tooltipEventArgs);
 
             this.visualHostTooltipService.show({
                 coordinates: tooltipEventArgs.coordinates,
+                isTouchEvent: false,
                 dataItems: tooltipInfo,
                 identities: selectionId ? [selectionId] : [],
-                isTouchEvent: false,
             });
         });
 
         selection.on("mouseout.tooltip", () => {
             this.visualHostTooltipService.hide({
-                immediately: false,
                 isTouchEvent: false,
+                immediately: false,
```

Comment:
Does it affect on visual behavior anyhow?

Reply:
Copied tooltipservicewrapper from 
https://github.com/microsoft/PowerBI-visuals-sampleBarChart/blob/dev/next/src/tooltipServiceWrapper.ts

This is the only change that seems to fix the tooltip issue.

Reply:
Correction: tooltip is still broken. I am working an that but I would like this change to go in since its affecting the customers